ufs_cmds.c

```diff
@@ -1176,6 +1177,9 @@ void print_command_help(char *prgname, int config_type)
 	case ERR_HIST_TYPE:
 		err_hist_help(prgname);
 	break;
+	case UIC_TYPE:
```

Comment:
I think you should, in this patch, introduce an empty "do_uic", otherwise it will be silently ignore.
 At this point, "do_uic" should return -ENOTSUPP or something, with the proper "not supported yet" message